### ISSUE TITLE
Uncaught (in promise) ChunkLoadError: Loading chunk 12 failed.

### ISSUE THREAD

**Issue by amithm7** [lelouch77/docusaurus-lunr-search#49]
This shows up after fixing #42.

Doesn't deploy to GitHub pages either. Is this a problem with lazy loading?

**Comment by lelouch77** [comment]
I don't think so, I tired the fix #42 in local, it doesn't created any such error, please verify it again

**Comment by amithm7** [comment]
Hi @lelouch77 . Thanks for replying, this isn't a problem that I had encountered in local build, but in production.

To reproduce, you could fork this repo at the commit https://github.com/celzero/docs/commit/1912973e05c899601545b2b938cae53fff72b2da and have it deploy to github pages.
